index.html

```diff
@@ -486,6 +486,21 @@ <h3>CONTRIBUTORS</h3>
             </div>
             <!-- COPY TILL HERE -->
 
+            <!-- COPY FROM HERE -->
+            <div class="col-md-4">
+                <div class="m-card">
+                    <div class="name">Brody Underwood</div>
+                    <div class="msg">Be the chaos you want to see in the world.</div>
+                    <div class="github container-fluid">
+                        <div class="row">
+                            <i class="fab fa-github"></i>
+                            <a href="https://github.com/BrodyUnderwood1">/Username</a>
```

Comment:
add your username in place of "/Username"

Reply:
lol whoops ty will do